fcl/protodunevd/g4/protodunevd_refactored_g4_stage2_pureAr.fcl

```diff
@@ -0,0 +1,69 @@
+#include "services_refactored_pdune.fcl"
```

Comment:
Could also inherit from another fhicl to simplify it and make maintenance easier.